.github/filters.yaml

```diff
@@ -0,0 +1,18 @@
+backend:
+  - 'packages/**/package.json'
+  - 'packages/**/server/**'
+  - 'packages/**/strapi-server.js'
+  - 'packages/providers/**'
+  - 'packages/utils/**'
+  - 'packages/generators/**'
+  - 'packages/cli/**'
+  - 'api-tests/**'
+  - 'packages/core/*/lib/**'
+  - 'packages/core/*/bin/**'
+  - 'packages/core/*/ee/**'
+frontend:
+  - 'packages/**/package.json'
+  - 'packages/**/admin/src/**'
+  - 'packages/**/strapi-admin.js'
+  - 'pacakges/core/helper-plugin/**'
```

Comment:
`packages` ?

Reply:
Yes since this works from root if you look at the github file system the first folder is packages

Reply:
Yes I meant there was a typo `pacakges` vs `packages`

Reply:
I am officially blind and dyslectic